.github/actions/spelling/expect.txt

```diff
@@ -686,3 +686,7 @@ YOURNAME
 yourregistry
 youtube
 squidfunk
+databranch
+ghtoken
+jgehrcke
+ghrs
```

Comment:
could you please add newline at the end of file to fix this warning

Reply:
sure, added